src/libtls/tls_server.c

```diff
@@ -757,6 +1134,32 @@ static status_t send_certificate(private_tls_server_t *this,
 	return NEED_MORE;
 }
 
+/**
+ * Send Certificate Verify
+ */
+static status_t send_certificate_verify(private_tls_server_t *this,
+							tls_handshake_type_t *type, bio_writer_t *writer)
+{
+	private_key_t *key;
+
+	key = lib->credmgr->get_private(lib->credmgr, KEY_ANY, this->server,
```

Comment:
Hm, didn't `select_suite_and_key()` select a private key and store it in `this->private`? (But also see the comments there regarding the selection process.)